src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_help.py

```diff
@@ -263,6 +263,11 @@
     short-summary: Start a job.
 """
 
+helps['ams job update'] = """
+    type: command
+    short-summary: Updates an existing job.
```

Comment:
The verb should be in its infinitive form: 'Update an existing job'

Reply:
done